modules/wpa_screenshot_capture/js/wpa-slider.js

```diff
@@ -1,32 +1,40 @@
-/**
- * Initializes the WPA comparison slider.
- */
-function wpaInitializeSlider() {
-  var images = document.querySelectorAll('.image-style-web-page-archive-full');
-  var ct = 0;
+/** global: jQuery */
+/** global: Drupal */
 
-  /**
-   * Callback function when image is loaded.
-   */
-  function imageReady() {
-    if (++ct >= 2) {
-      jQuery('.ba-slider').beforeAfter();
-    }
-  }
+(function ($, Drupal) {
+  Drupal.behaviors.wpaBeforeAfterSlider = {
+    attach: function (context, settings) {
+      var $modal = $('#drupal-modal');
+      var images = document.querySelectorAll('.image-style-web-page-archive-full');
```

Comment:
Although this PR is already approved, I want to point out the mixture of native document API usage and jQuery usage. I prefer native but would favor jQuery if it was the choice to make things more consistent. The mixture feels assembled, copy/pasty...

Reply:
Yeah I was thinking that while I was working on it. It should be minimal effort to switch so I’ll push that change up in a little bit.